pom.xml

```diff
@@ -45,19 +47,27 @@
 
   <distributionManagement>
     <repository>
-        <id>maven</id>
-        <name>maven-central</name>
-        <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+      <id>openhab-releases</id>
+      <name>Reposilite self-hosted openHAB Maven Release Repository</name>
+      <url>${oh.repo.releaseBaseUrl}</url>
     </repository>
+    <snapshotRepository>
+      <id>openhab-snapshots</id>
+      <name>Reposilite self-hosted openHAB Maven Snapshot Repository</name>
+      <url>${oh.repo.releaseBaseUrl}</url>
+    </snapshotRepository>
   </distributionManagement>
 
   <properties>
     <oh.scm.gitBaseUrl>https://github.com/openhab</oh.scm.gitBaseUrl>
-    <oh.repo.distBaseUrl>https://openhab.jfrog.io/openhab</oh.repo.distBaseUrl>
-    <oh.repo.releaseBaseUrl>https://openhab.jfrog.io/openhab</oh.repo.releaseBaseUrl>
-    <oh.repo.snapshotBaseUrl>${oh.repo.releaseBaseUrl}</oh.repo.snapshotBaseUrl>
+    <oh.repo.distBaseUrl>https://gitlab.com/tvision360/infrastructure</oh.repo.distBaseUrl>
+    <oh.repo.releaseBaseUrl>${maven.repo.url}/${maven.release.endpoint}</oh.repo.releaseBaseUrl>
```

Comment:
Dynamic repo configuration is useful for forking the application for enterprise use.